pandas/core/series.py

```diff
@@ -234,6 +243,10 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
                                        copy=copy)
                 elif copy:
                     data = data.copy()
+            elif isinstance(data, ExtensionArray):
```

Comment:
I don't really like this here, instead this should be handled in _sanitize_array (which will basically ignore this), otherwise you have indexes take different paths there than EA

Reply:
Yes, I think we're OK to remove it since `_sanitize_array` handles EA now. Probably just a leftover from an early round. Testing now.